src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -52,12 +52,6 @@ internal abstract partial class NamedTypeSymbol
 
             NamedTypeSymbol underlyingType = getTupleUnderlyingType(elementTypesWithAnnotations, syntax, compilation, diagnostics);
 
-            if (numElements >= ValueTupleRestPosition && diagnostics != null && !underlyingType.IsErrorType())
-            {
-                WellKnownMember wellKnownTupleRest = GetTupleTypeMember(ValueTupleRestPosition, ValueTupleRestPosition);
-                _ = GetWellKnownMemberInType(underlyingType.OriginalDefinition, wellKnownTupleRest, diagnostics, syntax);
```

Comment:
>_ = GetWellKnownMemberInType(underlyingType.OriginalDefinition, wellKnownTupleRest, diagnostics, syntax); [](http://example.com/codeflow?start=16&length=105)

Are we going to "intercept" the errors, that possibly used to be reported here, some where else?

Reply:
It seems to me that's not necessary - whoever is using the `Rest` member will report those errors if it's missing.

Reply:
> It seems to me that's not necessary ...

It sounds like the answer is "yes", the error will be reported by someone else. At least this is the hope. How confident you are and what is the basis for that level of confidence?

Reply:
> It sounds like the answer is "yes"

Yes, sorry, I misunderstood the question.

> How confident are you, and what is the basis for that level of confidence?

If someone wants to access the `Rest` member, they should do that via `WellKnownMember` enum in a way that reports diagnostics if that member is missing. I've looked through usages of `NamedTypeSymbol.GetTupleTypeMember` and they seem to be doing that where necessary.